### ISSUE TITLE
[BUG] Broken Arrow Meta Gradle plug-ins

### ISSUE THREAD

**Issue by serras** [arrow-kt/arrow-meta#926]
It seems that I've somehow broken the Gradle plug-ins for Arrow Meta in one of the PRs. Right now whaetever plug-in you try to apply, you are always welcome with:

```
e: java.lang.IllegalStateException: Generated sources output dir is not found: directory to locate sources
        at arrow.meta.phases.analysis.DefaultElementScopeKt.getOrCreateBaseDirectory(DefaultElementScope.kt:495)
        at arrow.meta.plugins.analysis.phases.PhasesKt$analysisPhases$1.invoke(Phases.kt:38)
        at arrow.meta.plugins.analysis.phases.PhasesKt$analysisPhases$1.invoke(Phases.kt:31)
        at arrow.meta.dsl.analysis.AnalysisSyntax$analysis$1.doAnalysis(AnalysisSyntax.kt:89)
```

Alas, I cannot find the source of the problem. My guess is that it's related to the introduction of different names for plug-ins, which was required to be able to apply more than one in the same project.

**Comment by raulraja** [comment]
@serras I'm looking at this today. Thanks.

**Comment by serras** [comment]
Closed by #927
